README.md

```diff
@@ -2,11 +2,37 @@
 
 [![Build Status](https://travis-ci.com/tomMoulard/fail2ban.svg?branch=main)](https://travis-ci.com/tomMoulard/fail2ban)
 
-This plugin is a small implementation of a fail2ban instance as a middleware
+This plugin is a small implementation of a Fail2ban instance as a middleware
 plugin for Traefik.
 
+## Middleware
+
+After installing the plugin, it can be configured through a Middleware, e.g.:
+
+```yml
+apiVersion: traefik.containo.us/v1alpha1
+kind: Middleware
+metadata:
+  name: fail2ban-test
+spec:
+  plugin:
+    fail2ban:
+      logLevel: DEBUG
+      blacklist:
+        ip: 127.0.0.1
+```
+
+Don't forget to add the middleware to your Route (and since middlewares are
```

Comment:
Don't you think that providing a full working example could be better? It could be placed in a `<details>` to be less annoying for people looking for the Plugin doc? WDYT ?